parsing/intermediate/dialects/defaultdialect/src/test/java/org/apache/calcite/test/DefaultDialectParserTest.java

```diff
@@ -27,4 +29,15 @@
   @Override protected SqlParserImplFactory parserImplFactory() {
     return DefaultDialectParserImpl.FACTORY;
   }
+
+  @Test void testCastFailsOnlyInDefaultDialect() {
```

Comment:
nit: You can remove the "OnlyInDefaultDialect" suffix here. Based on the file this test is defined in and the way the test inheritance is structured, that should be automatically implied.

Reply:
Fixed in https://github.com/googleinterns/calcite/pull/216/commits/4c4612dab93e39794b54dc478f3f52a64438567d